### DISCUSSION TITLE
The link https://github.com/pointfreeco/episode-code-samples/tree/main/0240-reliably-testing-async-pt3 is dead

### DISCUSSION THREAD

**Discussion by flockoffiles** [pointfreeco/episode-code-samples#144] [General]
Unfortunately the link https://github.com/pointfreeco/episode-code-samples/tree/main/0240-reliably-testing-async-pt3 is dead.

The link itself is present at the bottom of https://www.pointfree.co/episodes/ep240-reliable-async-tests
(under "Sample Code")

**Comment by stephencelis** [comment]
Should be pushed now, thanks!
